models/classes/export/implementation/CsvExporter.php

```diff
@@ -105,4 +115,9 @@ public function getFileExportResponse(
         $exportedString = $this->export($columnNames, false, $delimiter, $enclosure);
         return $this->preparePsrResponse($originResponse, $exportedString, self::FILE_NAME);
     }
+
+    private function prependBomCharSet(string $data): string
+    {
+        return chr(0xEF) . chr(0xBB) . chr(0xBF) . $data;
```

Comment:
Pretty sure you can use escape chars instead.
```suggestion
        return "\xEF\xBB\xBF{$data}";
```